data/fields/hazard_highway.json

```
@@ -0,0 +1,31 @@
+{
+    "key": "hazard",
+    "type": "combo",
+    "strings": {
+        "options": {
+            "animal_crossing": "Animal Crossing",
+            "bump": "Bump",
+            "children": "Children",
+            "curve": "Curve",
+            "curves": "Multiple Curves",
+            "cyclists": "Cyclists",
+            "dangerous_junction": "Dangerous Junction",
+            "dip": "Dip",
+            "falling_rocks": "Falling Rocks",
+            "frost_heave": "Frost Heave",
+            "horse_riders": "Horse Riders",
+            "ice": "Ice",
+            "landslide": "Landslide",
+            "loose_gravel": "Loose Gravel",
+            "low_flying_aircraft": "Low Flying Aircraft",
+            "pedestrians": "Pedestrians",
+            "queues_likely": "Queues Likely",
+            "school_zone": "School Zone",
+            "side_winds": "Side Winds",
+            "slippery": "Slippery",
+            "turn": "Turn",
+            "turns": "Reverse Turn"
+        }
+    },
+    "label": "Hazard"
```

```suggestion
    "label": "Hazard (Signed)"
```

Or

```suggestion
    "label": "Signed Hazard"
```

Had the same thought as @matkoniecz 
But this goes for all those values, right?